src/db.rs

```diff
@@ -199,8 +199,10 @@ pub async fn add_holoport_status(hs: HostStats, db: &Client) -> Result<(), ApiEr
         "holoportId": hs.holoport_id,
         "timestamp": hs.timestamp,
         "hposAppList": hpos_app_list,
+        "channelVersion": hs.channel_version,
+        "hposVersion": hs.hpos_version,
```

Comment:
Can I get some extra scrutiny on these lines to make sure this is okay to do?

Reply:
Yes, mongoDB has a flexible schema.

Reply:
But extending schema requires adding those new elements to the pipeline in a `list_available_hosts()` function above.
```
    let pipeline = vec![
        doc! {
            // only successful ssh results in last <cutoff> days
            "$match": {
                "sshStatus": true,
                "timestamp": {"$gte": cutoff_ms}
            }
        },
        doc! {
            // sort by timestamp, descending:
            "$sort": {
                "timestamp": -1
            }
        },
        doc! {
            "$group": {
                "_id": "$holoportId",
                "holoNetwork": {"$first": "$holoNetwork"},
                "channel": {"$first": "$channel"},
                "holoportModel": {"$first": "$holoportModel"},
                "sshStatus": {"$first": "$sshStatus"},
                "ztIp": {"$first": "$ztIp"},
                "wanIp": {"$first": "$wanIp"},
                "timestamp": {"$first": "$timestamp"},
                "hposAppList": {"$first": "$hposAppList"},
                "channelVersion": {"$first": "$channelVersion"},
                "hposVersion": {"$first": "$hposVersion"},
            }
        },
        doc! {
            "$project": {
                "_id": 0,
                "holoportId": "$_id",
                "holoNetwork": "$holoNetwork",
                "channel": "$channel",
                "holoportModel": "$holoportModel",
                "sshStatus": "$sshStatus",
                "ztIp": "$ztIp",
                "wanIp": "$wanIp",
                "timestamp":"$timestamp",
                "hposAppList": "$hposAppList",
                "channelVersion": {"$first": "$channelVersion"},
                "hposVersion": {"$first": "$hposVersion"},
              }
        },
    ];
```